### ISSUE TITLE
npm package is missing

### ISSUE THREAD

**Issue by mkhazov** [toolbox-tve/videojs-dvrseekbar#1]
```npm ERR! 404 Registry returned 404 for GET on https://registry.npmjs.org/videojs-dvrseekbar```

**Comment by alexchern** [comment]
Same for me :-(

**Comment by upsidedownmind** [comment]
Hi guys, it should be working now:

https://www.npmjs.com/package/videojs-dvrseekbar

**Comment by alexchern** [comment]
Hi there,

Thanks a lot. Now it is accessible..
I'v encountered some issues (trying to use it with browserify).
1. I had to add videojs to global window, otherwise it didn't work.
2. It also did't work properly ("Live" button is only visible on hover) then I tried to use it the standard way. I was able to make it work using browserify-shim though.
3. It mostly works (well, sometimes the slider extends well over the allowed area to the right) on Chrome and Firefox. It didn't work for me on Edge - the slider is not visible at all.

Thanks for your help,

Alex
